applications_tests/lethe-fluid-matrix-free/mms3d_fe1_lsmg.mpirun=2.output

```diff
@@ -0,0 +1,18 @@
+Running on 2 MPI rank(s)...
+   Number of active cells:       64
+   Number of degrees of freedom: 500
+   Volume of triangulation:      8
+
+*****************************
+Steady iteration:        1/2
+*****************************
+
+*****************************
+Steady iteration:        2/2
+*****************************
+   Number of active cells:       512
+   Number of degrees of freedom: 2916
+   Volume of triangulation:      8
+cells  error_velocity     error_pressure   
+   64 4.185618e-01    - 4.553725e-02     - 
+  512 1.099013e-01 1.93 1.771718e-01 -1.96  
```

Comment:
After our discussion, I feel it's more coherent here to do the test with a sufficient number of initial cells to show that there is at least a degree of convergence for both variables